cicecore/cicedynB/general/ice_init.F90

```diff
@@ -1641,13 +1649,18 @@ subroutine input_data
          write(nu_diag,1010) ' rotate_wind      = ', rotate_wind,' : rotate wind/stress to computational grid'
          write(nu_diag,1010) ' formdrag         = ', formdrag,' : use form drag parameterization'
          write(nu_diag,1000) ' iceruf           = ', iceruf, ' : ice surface roughness at atmosphere interface (m)'
-         if (trim(atmbndy) == 'default') then
-            tmpstr2 = ' : stability-based boundary layer'
+         if (trim(atmbndy) == 'constant') then
+            tmpstr2 = ' : boundary layer uses bulk transfer coefficients'
```

Comment:
I would change 'bulk' to 'constant' in this line.  If I remember correctly, the overall parameterization we use is considered a 'bulk' formulation, and we're just choosing whether the coefficients are constant or not.  Anyone please correct me if that's wrong.  (And yes I used 'bulk' in the original comment!)

Reply:
Changed / rewritten